src/main/java/ModelsTest/UserTest/UserDaoTest.java

```diff
@@ -0,0 +1,93 @@
+package ModelsTest.UserTest;
+
+import Models.User.DuplicateEmailPresentException;
+import Models.User.User;
+import Models.User.UserDao;
+import Models.User.UserNotFoundException;
+import jdk.nashorn.internal.ir.annotations.Ignore;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class UserDaoTest {
+
+    private final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+
+    @BeforeEach
+    void setUp() {
+        System.setOut(new PrintStream(outputStreamCaptor));
+    }
+
+    @AfterEach
+    void tearDown() throws IOException {
+        outputStreamCaptor.close();
+        // TODO: Revert changes made to database in unit test 'Given_ValidUser_Then_InsertUser'
+    }
+
+    @Ignore
+    @Test
+    void Given_ValidUser_Then_InsertUser() throws DuplicateEmailPresentException {
+        // Given
+        User user = new User(12, "Charlie", "Williams,", "charlie@uwe.ac.uk", new byte[16], new byte[16]);
+
+        // When
+        UserDao.insertUser(user);
+
+        // Then
+        assertEquals("User inserted", outputStreamCaptor.toString());
```

Comment:
We probably want to refactor DAO methods to return a bool - I feel like checking the output is a bit cursed lol